src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -48,16 +52,32 @@ private static string[] GetBuiltinExtensionAssemblies()
 
         public Type[] GetStartupTypes()
         {
-            IEnumerable<Type> startupTypes = GetExtensionsStartupTypes();
+            IEnumerable<Type> startupTypes = GetExtensionsStartupTypesAsync().ConfigureAwait(false).GetAwaiter().GetResult();
 
             return startupTypes
                 .Distinct(new TypeNameEqualityComparer())
                 .ToArray();
         }
 
-        public IEnumerable<Type> GetExtensionsStartupTypes()
+        public async System.Threading.Tasks.Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
```

Comment:
Same as above. Why did you change this to async? I do not see any async calls in this method

Reply:
have an async call below `string extensionBundlePath = await _extensionBundleManager.GetExtensionBundle();`